php/php.editor/tools/ASTPHP5Parser.cup

```diff
@@ -4310,6 +4312,19 @@ common_scalar:scalar
 :}
 ;
 
+/* PHP 8.1: New in initializer */
+static_scalar_value_with_class_instance ::=
+static_scalar_value:scalar
+{:
+    RESULT = scalar;
+:}
+
+| T_NEW:n class_name_reference:className ctor_arguments:ctor
+{:
+    RESULT = new ClassInstanceCreation(nleft, ctorright, className, ctor);;
```

Comment:
Nitpick: typo `;;`

Reply:
Oops, Will fix it with the next fix. Thanks!

Reply:
No problem :)